### DISCUSSION TITLE
I made a mirror for the builds :)

### DISCUSSION THREAD

**Discussion by accessiblepixel** [Aterfax/relax-intel-rmrr#22] [Announcements]
Hiya @Aterfax - Remember me saying I was thinking about setting up a mirror of all of the wonderful builds you've made? As a thank you to your effort (and those before you) I went and did it.

I already had all the (successful) 'artefacts' from the Github Actions still around, so figured it might be nice, especially now given some of those older artefacts have expired.

I've tried to follow a fairly sensible scheme for the layout.

**/releases/** is just a mirror of anything you've put as a release on github.

**/automatic-builds/** is a copy of the build artefacts from the Github Actions
I followed the naming scheme `<Year>/<Month>/<YYYY>-<MM>-<DD>_Build#<BuildNumberFromAction>_Kernel-<version>.zip` but they're just the same files that were created, just renamed to make them easier to differentiate. So for example, a build might look like this `2022/07/2022-07-02_Build#19_Kernel-5.15-39.1.zip`

**/tested/** is a folder that I can symlink to versions from the automatic ones that I've tested and that have worked on my hardware.

I've added a header with a bit of information so that people can find the project, the history, and tried to recognise those that have been involved in the history too. The link is https://mirror.jcx.life/intel-relax-rmrr/

I hope you like this, and again much appreciation from my side for keeping these patches up and running. If you've any feedback and suggestions, I would love to hear them. I hope you're having a good week :)

~ Jessica

**Comment by Aterfax** [comment]
Cracking, thanks for getting this setup! If you fancy chucking in a PR pointing the mirror out in the README I'll happily merge it!

**Comment by accessiblepixel** [comment]
I added the releases and last couple of automatic builds to it, haven't had chance to test the new ones yet but I made sure to mirror them.

I might do a pull onto the readme, but at first I wanna practice for a bit with getting them on there ^^,

**Comment by accessiblepixel** [comment]
Pulled the latest action results into the mirror (up to 2022-11-19). It's still a manual process for now, but they're mirrored. Still haven't tested a newer image yet, but I've almost got my test bed ready, so I can test the images before deploying them to my cluster :) ... less trial by fire :p

Hope you're doing good <3

**Comment by accessiblepixel** [comment]
Pulled the latest builds again, been a little while but finally got them up to date. I missed a couple since they'd expired before I could grab them, but almost all are there now as of today :)
